notes/2023/2023-11-30.md

```diff
@@ -0,0 +1,42 @@
+# 2023-11-30 ESLint TSC Meeting Notes
+
+## Transcript
+
+[`2023-11-30-transcript.md`](2023-11-30-transcript.md)
+
+## Attending
+
+* Nicholas C. Zakas (@nzakas) - TSC
+* Milos Djermanovic (@mdjermanovic) - TSC
+
+@nzakas moderated, and @sam3k took notes.
+
+## Topics
+
+### [v9.0.0 Status](https://github.com/orgs/eslint/projects/4/views/2) 
+
+| Feature | Status | Notes |
+| ------- | -- | -- |
+| [Change Request: Drop Node v12/v14/v16/v17/v19 support](https://github.com/eslint/eslint/issues/17595) | PR is ready for merge | @nzakas and @mdjermanovic will set status to ready for merge |
+| [Change Request: Remove all formatters except stylish, html, json, and json-with-meta](https://github.com/eslint/eslint/issues/17524) | Waiting on the last remaining changes from @mdjermanovic | |
+| [Change default rule schema to `schema: []` and drop support for function-style rules](https://github.com/eslint/eslint/issues/14709) | @mdjermanovic opened [draft PR](https://github.com/eslint/eslint/pull/17792) | |
+| [Change Request: stop running rules set to "warn" when `--quiet` is used](https://github.com/eslint/eslint/issues/16450) | Close to done; just making final changes. | |
+| [Omiting any file/dir positional parameter to CLI does not give an error](https://github.com/eslint/eslint/issues/14308) | Ready for review | @mdjermanovic will review soon |
+| [Implement Language Plugins](https://github.com/eslint/eslint/issues/16999) | Once [#17698](https://github.com/eslint/eslint/pull/17698) and [#143](https://github.com/eslint-community/eslint-plugin-n/issues/143) are merged, we can move this one to ready to merge  | |
+| [Implement Flat Config](https://github.com/eslint/eslint/issues/13481) | **Blocked** | Blocked by work left on getting the rule tests switched over to use `FlatRuleTester`. Once that's merged, @nzakas will start on the docs. Also, we won't rename files in this PR. It will be handle separately. |
```

Comment:
Comments are disabled for this issue so won't be able to leave an update.

Reply:
This is what I see:
<img width="964" alt="Screenshot 2023-12-05 at 7 52 32 PM" src="https://github.com/eslint/tsc-meetings/assets/180152/237c6d69-1812-4ca8-ad62-b6e6f9d5f70f">

Btw, it is the same case with Discord's tsc-meeting channel. I cannot comment or react to a comment. I can only react to someone's else reactions. In case it seems I'm very quiet when I do attend. :)

Reply:
Discord is intentionally set up that way. :)

Reply:
Okay, so you have "triage" permissions on the repo and apparently that wasn't enough. I've bumped that up to "write" permissions. Can you see if that works?